flag_bool.go

```diff
@@ -19,6 +19,7 @@ type BoolFlag struct {
 	DefaultText string
 	Destination *bool
 	HasBeenSet  bool
+	Category    string
```

Comment:
@meatballhat How would this tie-in with your generator code ?

Reply:
If all the flag types defined in this repo have `Category` fields, then it'd be part of the struct template. Or did you mean the potential merge conflict? (or something else?)